src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/androids/MinerAndroid.java

```diff
@@ -111,8 +109,7 @@ protected void moveAndDig(Block b, BlockMenu menu, BlockFace face, Block block)
                 }
 
                 // We only want to break non-Slimefun blocks
-                if (!BlockStorage.hasBlockInfo(block)) {
-                    block.getWorld().playEffect(block.getLocation(), Effect.STEP_SOUND, block.getType());
+                if (!BlockStorage.hasBlockInfo(block)) {   
```

Comment:
Remove trailing spaces
```suggestion
                if (!BlockStorage.hasBlockInfo(block)) {
```